cmd/flexible-informer-gen/generators/informer.go

```diff
@@ -163,6 +165,12 @@ func NewFiltered$.type|public$Informer(client $.clientSetInterface|raw$$if .name
 				unstructuredResult := &unstructured.UnstructuredList{}
 				err := client.$.group$$.version$().RESTClient().Get().$if .namespaced$Namespace(namespace).$end$Resource($.toLower$("$.type|publicPlural$")).VersionedParams(&options, $.schemes|raw$).Timeout(timeout).Do(context.TODO()).Into(unstructuredResult)
 				if err != nil {
+					if $.isNotFound$(err) {
+						logs.warnf("The version of CRD for this kind is too old/new or not existing. All the resources will be ignored", 
```

Comment:
"All the resources will be ignored" could possibly make users think *_all_* resources will be ignored (even ones for which the CRD is found). The other suggestion is just wordsmithing that you're free to ignore. The same suggestion goes for the equivalent message in the Watch function.
```suggestion
						logs.warnf("The expected version of CRD for this kind was not found; it may be too old/new or not existing. All the resources for this kind will be ignored", 
```